Websites/webkit.org/wp-content/plugins/hyperlight/hyperlight/preg_helper.php

```diff
@@ -115,10 +115,9 @@ function preg_merge($glue, array $expressions, array $names = array()) {
             /x';
             $sub_expr = preg_replace_callback(
                 $backref_expr,
-                create_function(
-                    '$m',
-                    'return $m[1] . "\\\\" . ((int)$m[2] + ' . $capture_count . ');'
-                ),
+                function($m) {
+                    return $m[1] . "\\\\" . ((int)$m[2] + $capture_count);
```

Comment:
I assume that you tested to confirm that all the backslashes are actually still needed. Naively, it seems like going away from string representation would need less escaping.

Reply:
This is tested and proven working on a on a dev instance.

Reply:
This is tested and proven to work as expected on a dev instance.